### ISSUE TITLE
Command "SFDX: Diff File Against Org" shows up in non-SFDX projects

### ISSUE THREAD

**Issue by muenzpraeger** [forcedotcom/salesforcedx-vscode#1600]
### Summary

When using the command palette the command "SFDX: Diff File Against Org" shows up in non-SFDX projects.

### Steps To Reproduce:

1.  Open any folder in VS Code.
1.  Type *SFDX*. The command shows up.

### Expected result

The command should only shop up in SFDX projects, and when a supported file is open.

### Actual result

The command always shows up.

**Comment by muenzpraeger** [comment]
Org based is also SFDX.

Let me rephrase - it should only show on projects that have a `sfdx-project.json` file.

**Comment by lcampos** [comment]
This got fixed in version `46.13.0` of the extensions.

**Comment by ruwanthalk** [comment]
This issue had when I try to diff the code to org
Then found I had to install the diff first by `sfdx plugins:install @salesforce/sfdx-diff`
